arcgis-ios-sdk-samples/Scenes/Create terrain surface from a local tile package/README.md

```diff
@@ -0,0 +1,50 @@
+# Create terrain from a local tile package
+
+Set the terrain surface with elevation described by a local tile package.
+
+![Create terrain from a local tile package](create-terrain-from-a-local-tile-package.png)
+
+## Use Case
+
+The terrain surface is what the basemap, operational layers, and graphics are draped on. The tile package must be a LERC (limited error raster compression) encoded TPK. Details on creating these are in the [ArcGIS Pro documentation](https://pro.arcgis.com/en/pro-app/help/sharing/overview/tile-package.htm).
+
+Terrain can be loaded offline from tile packages (.tpk).
+
+## How it works
+
+1. Create an `ASScene` and add it to a `SceneView`.
+2. Create an `AGSArcGISTiledElevationSource` with the path to the local tile package.
+3. Add this source to the scene's base surface: ```
+surface.elevationSources.append(elevationSource)
+scene.baseSurface = surface```.
+
+## Relevant API
+
+* AGSArcGISTiledElevationSource
+* Surface
+
+## About the data
+
+This terrain data comes from Monterey, California.
+
+## Offline data
+
+1. Download the data from [ArcGIS Online](https://arcgisruntime.maps.arcgis.com/home/item.html?id=cce37043eb0440c7a5c109cf8aad5500).
```

Comment:
Does the user have to download the data this way for this sample? I have seen samples where this is done automatically.